src/Octokit.Webhooks/Converter/DateTimeStringConverter.cs

```diff
@@ -0,0 +1,40 @@
+namespace Octokit.Webhooks.Converter
+{
+    using System;
+    using System.Globalization;
+    using System.Text.Json;
+    using System.Text.Json.Serialization;
+
+    public class DateTimeStringConverter : JsonConverter<string>
```

Comment:
Can you add a couple of unit tests for this class?

Reply:
Yeah sure, will do after we resolve all other questions :)

Reply:
Apologies but I'm a little confused - I can't seem to find any tests at all in this repo. I can see a test csproj with lots of resources in, but no tests. Running the documented test command locally appears to find no tests too, as did the GitHub Action for my branch. https://github.com/octokit/webhooks.net/runs/4799068273?check_suite_focus=true#step:6:20

Reply:
Good point. It's a little hypocritical of me to ask for tests, but I haven't actually written any myself.

I'm in the middle of writing some now, and I hit this exact issue with datetime types 🤦 
Let's forget about the tests for now. I'll scaffold the test setup in a separate PR, and one of us can create some tests for this specific class afterwards.

Reply:
> one of us can create some tests for this specific class afterwards.

Great, sounds like a plan :)